ZLocation/ZLocation.psm1

```diff
@@ -149,19 +141,8 @@ Register-PromptHook
 
 Set-Alias -Name z -Value Set-ZLocation
 
-# Create a list of aliases for Set-Zlocation, and add Set-Zlocation
-$szlAliases = @(
-    foreach ($alias in Get-Alias -Definition Set-ZLocation) {
-        [regex]::Escape($alias.Name)
-    }
-
-    'Set-ZLocation'
-)
-
-
-# Make a regex to match starting a string with an alias followed by a space
-$TabExpansionRegex = '^('+$($szlAliases -join '|')+') '
-
 Export-ModuleMember -Function @('Set-ZLocation', 'Get-ZLocation', 'Pop-ZLocation', 'Remove-ZLocation') -Alias z
 # export this function to make it accessible from prompt
 Export-ModuleMember -Function Update-ZLocation
+
+export-modulemember -function Find-Matches
```

Comment:
new line + casing for `export-modulemember`

Reply:
Whoops, this change was a mistake.  I'll revert it.